appsec-kit-starter/src/main/java/com/vaadin/appsec/service/AppSecDevToolsPlugin.java

```diff
@@ -43,4 +42,24 @@ public boolean handleMessage(String command, JsonObject data,
         devToolsInterface.send("command-received", data);
         return true;
     }
+
+    private void sendScanResult(int vulnerabilityCount,
+            DevToolsInterface devToolsInterface) {
+        LOGGER.info("Scan completed");
+        sendData("appsec-kit-scan", vulnerabilityCount, devToolsInterface);
+        LOGGER.info(
+                "Vulnerabilities sent to the client: " + vulnerabilityCount);
+    }
```

Comment:
This is a one-liner method, no need for it if we move logging into `sendData`.

Reply:
The goal is with the `sendScanResult` method to log only when a "real" scan happens and not on every `handleConnect` method call (e.g. page refresh, new window and new tab). The `refreshScanResult` method's purpose is to keep the message with the vulnerability count (basically send it again) in the AppSec Kit tab after the page is refreshed or new window or tab is opened.

Reply:
I understand the reason now, thanks! The idea to log the message only upon scan completion makes sense, but the method anyway is used only in one place (the lambda) so I'd still put it back there: since it does not have nested blocks or complex structure, having a separate method does not improve readability but instead force the reader to look for the method 👀

Reply:
Also, I'd switch to `debug` logging for this messages. WDYT?

Reply:
Also, based on the new scan event listener tracking, I removed the `appsec-kit-refresh` command to always use the `appsec-kit-scan`.